.github/CONTRIBUTING.md

```diff
@@ -37,6 +37,15 @@ Some hints:
  * Models sent/received by the controllers are defined in [/shared/models](/shared/models) directory
 
 
+## Improve the website
+
+PeerTube's website is [joinpeertube.org](https://joinpeertube.org), where people can learn about the project and how it works – note that it is not a PeerTube instance, but more the “marketing website”.
```

Comment:
I wouldn't use "marketing".

Reply:
That is usually called a landing page.

Reply:
I just checked and Mastodon says homepage. I think it's a good fit. 🙂